src/components/DayPicker.jsx

```diff
@@ -409,6 +419,30 @@ export default class DayPicker extends React.Component {
     });
   }
 
+  onMonthChange(currentMonth) {
+    // Translation value is a hack to force an invisible transition that
```

Comment:
I couldn't figure out a better way to do this, but that might have been due to my inexperience with React more than anything